roofit/roofitcore/src/RooHistPdf.cxx

```diff
@@ -217,7 +217,7 @@ Double_t RooHistPdf::evaluate() const
     }
   }
 
-  Double_t ret = _dataHist->weight(_histObsList, _intOrder, !_unitNorm, _cdfBoundaries);
+  Double_t ret = _dataHist->weightFast(_histObsList, _intOrder, !_unitNorm, _cdfBoundaries);
 //  cout << "RooHistPdf::evaluate(" << GetName() << ") ret = " << ret << " ";
 //  cout << _histObsList[0] << " ";
 //  _histObsList[0]->Print("");
```

Comment:
Kill useless code when you get close to it.

Reply:
Good idea